lib/main.dart

```diff
@@ -57,6 +66,56 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   int _counter = 0;
 
+  /////////////////////////////////////////////
+  // Создаём плеер и управление плейером
+  // Create a [Player] to control playback.
+  late Player player = Player();
+  // Create a [VideoController] to handle video output from [Player].
+  late VideoController controller = VideoController(player);
+
+  // Выбранный файл-видео
+  late File? _file_video;
+  late File? _file_sub;
+
+  void getFileVideo() async {
+   FilePickerResult? result = await FilePicker.platform.pickFiles(
+    type: FileType.video,
+    //allowedExtensions: ['mp4', 'mov', 'avi', 'wmv'],
+   );
+ 
+   if (result != null) {
+    String _videofilePath = result.files.single.path!;
+    print(_videofilePath);
+    player.open(Media(_videofilePath));
+     setState(() {});
```

Comment:
Как я сразу не заметил - кажется, эта строка слегка вне стандартного формата кода. Рекомендую запустить `dart format` для кода тогда, или же можем предложить определённый стиль кода для нашего проекта.
```suggestion
    setState(() {});
```